common-library/src/main/java/com/yas/commonlibrary/csv/BaseCsv.java

```diff
@@ -0,0 +1,5 @@
+package com.yas.commonlibrary.csv;
+
+public class BaseCsv {
+
```

Comment:
I think at least we should have some common fields for this class , for example id. Then remove id in children classes 
@CsvColumn(columnName = "id")
 private Long id;